.plzconfig

```diff
@@ -1,5 +1,5 @@
 [Please]
-version = 17.0.0-beta.6
+version = 17.4.0-beta.10
```

Comment:
Version bump required for the new `get_entry_points` built-in function in Please.